android/app/src/main/java/com/gdgnitsurat/quickshare/utils/FirebaseUtil.kt

```diff
@@ -10,16 +14,31 @@ object FirebaseUtil {
 
     private var firebaseAuth: FirebaseAuth = FirebaseAuth.getInstance()
     private var firebaseUser: FirebaseUser? = firebaseAuth.currentUser
+    private val database: FirebaseDatabase = FirebaseDatabase.getInstance()
+    private var deviceid :String = ""
 
-    fun addCurrentUserToFirebaseDatabase() {
-        val database: FirebaseDatabase = FirebaseDatabase.getInstance()
-        val databaseRef: DatabaseReference = database.reference
-        val user = User(firebaseAuth.currentUser?.displayName, firebaseAuth.currentUser?.email)
-        databaseRef.child("users").child(firebaseAuth.currentUser?.uid).setValue(user)
+    fun addCurrentUserToFirebaseDatabase(deviceID:String) {
+        val databaseRef: DatabaseReference = database.getReference("users")
+        val user = User(firebaseAuth.currentUser?.displayName, firebaseAuth.currentUser?.email, deviceID)
```

Comment:
Instead of storing deviceID in users -> devices node. Store it in addDeviceToFirebaseDatabase()

Reply:
We require deviceID at both the places

Reply:
I'm trying to say store deviceID at all the places using one function only
like,

addDeviceIDToFirebaseDatabase(){
        //add to users -> devices -> dId
       //add to devices -> dID
}